ExadelTimeTrackingSystem/ExadelTimeTrackingSystem.Data/Repositories/TaskRepository.cs

```diff
@@ -21,5 +20,35 @@ public Task<List<Models.Task>> GetTasksOnDateAsync(DateTime date)
             var filter = filterBuilder.Eq(d => d.Date, date.Date);
             return GetCollection<Models.Task>().Find(filter).ToListAsync();
         }
+
+        public Task DeleteTaskAsync(Guid id)
+        {
+            var filterBuilder = Builders<Models.Task>.Filter;
+            var filter = filterBuilder.Eq(d => d.Id, id);
```

Comment:
in lambda expression it's comfy to name this letter as first letter of model, so that it's clear from first view with which input type we work. E.g. task - t, project - p, user - u